ws-client/src/transport.rs

```diff
@@ -276,7 +283,9 @@ impl<'a> WsTransportClientBuilder<'a> {
 		}
 
 		// If the handshake succeeded, return.
-		let (sender, receiver) = client.into_builder().finish();
+		let mut builder = client.into_builder();
+		builder.set_max_message_size(self.max_payload_size);
+		let (sender, receiver) = builder.finish();
 		Ok((Sender { inner: sender }, Receiver { inner: receiver }))
```

Comment:
Do you think we should have a "constructor" of some sort for `(Sender, Receiver)`?

Reply:
It's already is part of a builder pattern but I can remove those coupled helper methods.